src/axolotl/utils/trainer.py

```diff
@@ -434,6 +442,9 @@ def prepare_optim_env(cfg):
             stage = deepspeed_config.get("zero_optimization", {}).get("stage", None)
         setup_deepspeed_env(cfg, stage=stage)
 
+    if cfg.torch_compile:
+        setup_torch_compile_env(cfg)
```

Comment:
```suggestion
    setup_torch_compile_env(cfg)
```

There is already a condition check within the function. This outer one is not necessary